example/composition-k8s.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apiextensions.crossplane.io/v1
```

Comment:
Can this example be flushed out with a README, render command, XR/XRD so that it can be run locally and in-cluster?

Reply:
done!